euphonic/plot.py

```diff
@@ -166,11 +163,9 @@ def plot_1d(spectra: Union[Spectrum1D,
 
     gridspec_kw = _get_gridspec_kw(spectra)
     fig, subplots = plt.subplots(1, len(spectra), sharey=True,
-                                 gridspec_kw=gridspec_kw)
-    if not isinstance(subplots, np.ndarray):  # if only one subplot
-        subplots = np.array([subplots])
+                                 gridspec_kw=gridspec_kw, squeeze=False)
```

Comment:
Yeah, I was finding it slightly frustrating to test the logic and then realised this would be _way_ cleaner 😄 

I might just get into the habit of always using `squeeze=False` now for non-interactive sessions.